### PR TITLE
Orientdb integration

### PR DESCRIPTION
**Pull request by Winwardo**

### CODE REVIEW

File: src/shared/utilities.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+  /**
+   * Converts an object of style {'a': {'b': c}, 'd': {'e': f}} to [{'b': c}, {'e': f}]
+   * @param {Object} givenObject Any object.
+   * @returns {Array}
+   */
+export const flattenObjectToArray = (givenObject) => {
+  const result = [];
+  for (const key in givenObject) {
```
Comment (Winwardo) [inline src/shared/utilities.js]: Codacy Issue found: [The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2164426105/issues/source?bid=3148782&fileBranchId=3160401#l8)

File: src/server/index.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          console.log('TWEET FINDING')
+          console.log('' + tweetRecord['@rid'])
+
+
+          db.query('TRAVERSE in FROM (SELECT in("TWEETED") from :id)',
+            {'params':{'id': tweetRecord['@rid']}}
+        )
+            .then((tweeterRecord_) => {
+              const tweeterRecord = tweeterRecord_[0];
+              console.log("THEN")
```
Comment (Winwardo) [inline src/server/index.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2164912261/issues/source?bid=3148782&fileBranchId=3160401#l28)

Comment (Winwardo) [inline src/server/index.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2164912261/issues/source?bid=3148782&fileBranchId=3160401#l28)

File: src/shared/utilities.js (hunk 1 above)
Comment (Winwardo) [inline src/shared/utilities.js]: Codacy Issue found: [The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2164912266/issues/source?bid=3148782&fileBranchId=3160401#l8)

Comment (Winwardo) [inline src/shared/utilities.js]: Codacy Issue found: [The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2164912266/issues/source?bid=3148782&fileBranchId=3160401#l8)

**Comment by Winwardo** [comment]
@VasilyShelkov Ready for comments and merging - please test on your machine

**Comment by VasilyShelkov** [comment]
Decided with orientDB and integrated into project, see #3

**Pull request merged**